openslides_backend/action/actions/projector_countdown/create.py

```diff
@@ -18,7 +18,12 @@ class ProjectorCountdownCreate(CreateAction):
     model = ProjectorCountdown()
     schema = DefaultSchema(ProjectorCountdown()).get_create_schema(
         required_properties=["meeting_id", "title"],
-        optional_properties=["description", "default_time"],
+        optional_properties=[
+            "description",
+            "default_time",
+            "used_as_poll_countdown_meeting_id",
```

Comment:
Is it possible to give these values from outside (Can a client pass data in there)?

Reply:
So it is more clear, what happens.

Reply:
A client should not be able to set these fields. What happens, if a client sends these fields now? Please add tests to verify that a client cannot set these fields.

If this is not possible: You must create a new internal action just for this purpose. @r-peschke did this, too, if I remember correctly.

Reply:
Yes, you'll need a seperate internal action for this or else these values can be set from outside.

Reply:
But this shouldn't even be necessary: description and default_time are correct (see https://github.com/OpenSlides/OpenSlides/wiki/projector_countdown.create) and the relation fields can just be set on the meeting instead of on the countdown. If you need an example, the group relations (`admin_group_id` and `default_group_id`) are set in the same fashion.

Reply:
Set the meeting *_countdown_id s with action_results in the instance.